SearchInRotatedSortedArray.java

```diff
@@ -0,0 +1,92 @@
+//Time Complexity : O(log(n)) Not sure
```

Comment:
You will visit all items in the list at least once in case of sorted array where the TC will be `O(n)`.